aspnetcore/signalr/get-started-signalr-core/sample/Hubs/ChatHub.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using System;
+using System.Threading.Tasks;
+
+namespace SignalRChat.Hubs
+{
+    public class ChatHub : Hub
+    {
+       public Task Send(string user, string message)
+        {
+            string timestamp = DateTime.Now.ToShortTimeString();            
+            return Clients.All.InvokeAsync("Send", timestamp, user, message);
```

Comment:
Should this be `SendAsync`? I get intellisense for SendAsync for `1.0.0-preview1-final`:

![image](https://user-images.githubusercontent.com/328122/36812025-b56633a6-1cc7-11e8-9248-d39676f7d42c.png)

Reply:
Indeed. It is SendAsync in preview 1. I missed that. Thanks @tugberkugurlu